app/assets/javascripts/admin_legacy/views/promotional_feature_items/_form.js

```diff
@@ -0,0 +1,49 @@
+(function () {
+  'use strict'
+  window.GOVUK = window.GOVUK || {}
+
+  var promotionalFeatureItemsForm = {
+    init: function init (options) {
+      this.form = document.querySelector(options.selector)
+      this.toggleImageAndYoutubeFieldsVisibility()
+      this.addRadioButtonEventListeners()
+    },
+
+    toggleImageAndYoutubeFieldsVisibility: function toggleImageAndYoutubeFieldsVisibility () {
+      var form = this.form
+      var checked = form.querySelector('fieldset input[type=radio]:checked')
```

Comment:
Bit of a nitpick, but this seems a bit too permissive as it'd catch any future radios. Could we put a class on the fieldset?